src/workerd/server/workerd.capnp

```diff
@@ -601,6 +601,9 @@ struct Worker {
     # workerd uses SQLite to back all Durable Objects, but the SQL API is hidden by default to
     # emulate behavior of traditional DO namespaces on Cloudflare that aren't SQLite-backed. This
     # flag should be enabled when testing code that will run on a SQLite-backed namespace.
+
+    enableDocker @5 :Bool;
```

Comment:
Maybe group the config into a `struct DockerConfig` ?
The next thing that you will need to pass is the container name and there should be more config later (List of ports, max instances, ...)

Reply:
@mikenomitch what options do we need to support? Isn't the container name dependent to the durable object id?